clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -121,15 +146,15 @@ export class CopybookDownloadService implements vscode.Disposable {
                 switch (e.type) {
                     case Type.InvalidCredentials:
                         errorMessage = INVALID_CREDENTIALS_ERROR_MSG.replace(PROFILE_NAME_PLACEHOLDER, element.profile);
+                        this.locked = true;
+                        this.queue.clean();
```

Comment:
From telemetry POV could be an interesting event to collect I suppose

Reply:
Make sense.